app/src/main/res/drawable/icon_search_white_24dp.xml

```diff
@@ -1,4 +1,4 @@
-<vector android:height="24dp" android:tint="#FFFFFF"
+<vector android:height="24dp" android:tint="#FFFFFF" android:autoMirrored="true"
```

Comment:
This should not be reversed. Only icons which represent direction should be reversed.

Also reformat this file.

Reply:
Reverted and reformatted.